account_global_discount_amount/tests/test_account_global_discount_amount.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests.common import SavepointCase
+
+
+class TestAccountGlobalDiscountAmount(SavepointCase):
```

Comment:
You can reuse this : https://github.com/odoo/odoo/blob/14.0/addons/account/tests/common.py

Reply:
can you add a test where you have 3 lines (one with a taxe A and then 2 with the taxe B)

Reply:
@sebastienbeau , it's done for the test where you have 3 lines (one with a taxe A and then 2 with the taxe B)